styles.css

```diff
@@ -0,0 +1,106 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+  font-family: "Poppins", sans-serif;
+}
```

Comment:
- [ ] Nice one with the entire layout. Kindly try to adjust the contents from left and from the right to match the figma design. 😉 

- Your design
<img width="259" alt="image" src="https://user-images.githubusercontent.com/41163571/234715361-86e518a8-d1fa-408e-89dd-53ec61f2f6a7.png">

- Figma design
<img width="236" alt="right" src="https://user-images.githubusercontent.com/41163571/228056013-322e1f2c-93c1-4f1b-b7e0-46a3b6a945c5.png">